tests/cypress/features/blocks.feature

```diff
@@ -6,84 +6,84 @@ Feature: Blocks
     Given I am on blocks page
     And I wait 1 seconds
 
-  @basic
-  Scenario: Load all blocks and load more blocks
-    Then I should see 20 blocks in table
-    When I click on showMoreBlocksBtn
-    And I wait 1 seconds
-    Then I should see 40 blocks in table
+  # @basic
+  # Scenario: Load all blocks and load more blocks
+  #   Then I should see 20 blocks in table
+  #   When I click on showMoreBlocksBtn
+  #   And I wait 1 seconds
+  #   Then I should see 40 blocks in table
```

Comment:
I think most of the commented e2e tests can still work / tweaked a little bit since MSW returns a valid response

Reply:
@eniolam1000752 There might still be changes on the service end, which means that some of the mock data might be changed again. So it's best to temporarily leave the e2e tests in the current state